### PR TITLE
CSP-776 Rename Added field to AddedDate in the Attendance table

### PR DESCRIPTION
**Pull request by SawNawDfE**

### CODE REVIEW

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.refactorlog
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.refactorlog:1]: Delete the refactorlog

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj [hunk 1]
```diff
@@ -92,4 +92,7 @@
   <ItemGroup>
     <PreDeploy Include="Scripts\Script.PreDeployment1.sql" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj:95]: Delete the refactor log

**Review by DevenKShah — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.refactorlog
Comment (SawNawDfE) [reply src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.refactorlog:1]: Done

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj (hunk 1 above)
Comment (SawNawDfE) [reply src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj:95]: Done

**Review by MarkFCain — APPROVED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**